homeassistant/auth/auth_store.py

```diff
@@ -178,6 +195,31 @@ def __init__(self, hass: HomeAssistant) -> None:
 
         return found
 
+    @callback
+    def async_mutate_refresh_token(
```

Comment:
We only use `refresh_token` as a container to save client information in this case. It never returns back to user. But I am imaging there is "refresh" or "regenerate" button in UI to allow user get a new access token, at the same time revoke previous issued

Back to the above comment in, if we have 1:m for client_id and refresh_token, then this enforce is unnecessary, otherwise, we still need a method to either recreate refresh_token or mutate jwt_key

Reply:
I think it's easier to solve the regenerate case by removing old refresh token and generate a new refresh and access token with same client id/icon. `auth_store` is already growing a lot and this should be kept as simple as possible.

Reply:
So I suggest we do without this method.